creator-node/src/services/stateMachineManager/stateMonitoring/findSyncRequests.jobProcessor.js

```diff
@@ -77,16 +86,67 @@ module.exports = async function ({
     } else if (userDuplicateSyncReqs?.length) {
       duplicateSyncReqs = duplicateSyncReqs.concat(userDuplicateSyncReqs)
     } else if (userErrors?.length) errors = errors.concat(userErrors)
+
+    // Increment total counters for the user's 2 secondaries so we can report an aggregate total
+    const { primary } = user
+    if (!primaryToSecondaryToResultCountsMap[primary]) {
+      primaryToSecondaryToResultCountsMap[primary] = {}
+    }
+    for (const [secondary, resultForSecondary] of Object.entries(
+      userResultsBySecondary
+    )) {
+      if (!primaryToSecondaryToResultCountsMap[primary][secondary]) {
+        primaryToSecondaryToResultCountsMap[primary][secondary] = {}
+      }
+      if (
+        !primaryToSecondaryToResultCountsMap[primary][secondary][
+          resultForSecondary
+        ]
+      ) {
+        primaryToSecondaryToResultCountsMap[primary][secondary][
+          resultForSecondary
+        ] = 0
+      }
+      primaryToSecondaryToResultCountsMap[primary][secondary][
+        resultForSecondary
+      ]++
+    }
   }
 
+  // Map the result of each findSyncs call to metrics for the reason a sync was found / not found
+  for (const [primary, secondaryToResultCountMap] of Object.entries(
+    primaryToSecondaryToResultCountsMap
+  )) {
+    for (const [secondary, resultCountMap] of Object.entries(
+      secondaryToResultCountMap
+    )) {
+      for (const [labelValue, metricValue] of Object.entries(resultCountMap)) {
+        metricsToRecord.push(
+          makeGaugeIncToRecord(
+            MetricNames.FIND_SYNC_REQUEST_COUNTS_GAUGE,
+            metricValue,
+            { result: labelValue }
+          )
+        )
+
+        // Log so we can find the primary+secondary for each result, but don't spam logs with the default result
+        if (labelValue !== 'not_checked') {
+          logger.info(
+            `Incrementing gauge for metric ${MetricNames.FIND_SYNC_REQUEST_COUNTS_GAUGE} from primary=${primary} to secondary=${secondary} with result=${labelValue}`
```

Comment:
nit - idt this log should be about "incrementing gauge for metric <,etric>" but rather "Recorded findSyncRequests step for ..." (feel free to rephrase